index.html

```diff
@@ -1,91 +1,75 @@
 <!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel ="stylesheet" href ="Currency.html"/>
-    <script src="Currency.JS"></script>
-    <title>Currency Converter App</title>
-</head>
-<body>
-    
-	<form id="currency_form">
-        <!-- Currency Converter -->
-        <h1 class="heading text-center display-2">
-            Currency Converter
-        </h1>
-        <hr>
-        
-     <div class="container">
-            <div class="main">
-                <div class="form-group">
-                    <label for="amount">
-                        Amount to Convert :
-                    </label>
-                    <input type="text" class="form-control searchBox" placeholder="0.00" id="oamount">
-                </div>
-                <div class="row">
-                    <div class="col-sm-6">
-                        <div class="input-group mb-3">
-                            <div class="input-group-prepend">
-                                <span class="input-group-text">From</span>
-                            </div>
-                        
-                            <select id="from-currency-dropdown">
-							<option value="">Select One …</option>
-							</select>
-							<label>Currency Code:</label>
-									<label id="selected-from-currency-label"></label>
-                        </div>
-                    </div>
-     
-                    <div class="col-sm-6">
-                        <div class="input-group mb-3">
-                            <div class="input-group-prepend">
-                                <span class="input-group-text">To</span>
-                            </div>
-                            <select id="to-currency-dropdown">
-							<option value="">Select One …</option>
-							</select>
-                        </div>
-                    </div>
-                </div>
-				
-				<br/>
-<br/>
-     
-                <div class="text-center">
-                    
-     
-                    <!-- convert button -->
-                    <button class="btn btn-primary convert m-2" type="submit">
-                        Convert
-                    </button>
-                    <!-- reset button -->
-                    <button class="btn btn-primary m-2" onclick="clearVal()">
-                        Reset
-                    </button>
-                </div>
-     
-            </div>
-            <div calss="text-center2">
-                <p>Exchange rate on: <label id="datetime-label"/></p>
-                
-            </div>
+<html>
+  <head>
+    <link rel="stylesheet" href="styles.css">
+    <title>Currency Conversion</title>
+  </head>
+  <body onload="populateCurrencyDropdown()">
```

Comment:
Avoid adding binding events with HTML attributes. Prefer using `addEventListener` in JS. I [wrote about that](https://github.com/SathiC1990/HYF-Currency-Converter/pull/1#discussion_r1670895770) last week already.

Reply:
Done.